src/test/java/com/google/api/codegen/protoannotations/testdata/java_library_no_gapic_config.baseline

```diff
@@ -822,7 +822,7 @@ public class LibraryServiceClient implements BackgroundResource {
    * Sample code:
    * <pre><code>
    * try (LibraryServiceClient libraryServiceClient = LibraryServiceClient.create()) {
-   *   BookOneOfName name = DeletedBookName.of();
+   *   BookName name = BookName.of("[SHELF_ID]", "[BOOK_ID]");
```

Comment:
This is more accurate because there is a file-level Resource defined as `Book`, and the request object `MoveBookRequest` has a 
```
(google.api.resource_reference) = "google.example.library.v1.Book"
```